src/engine/openspaceengine.cpp

```diff
@@ -132,7 +133,7 @@ OpenSpaceEngine::OpenSpaceEngine(
     , _settingsEngine(new SettingsEngine)
     , _timeManager(new TimeManager)
     , _downloadManager(nullptr)
-    , _parallelConnection(new ParallelConnection)
+    , _parallelConnection(std::make_unique<ParallelConnection>())
```

Comment:
`new ParallelConnection` would be fine as well..  The old version calls the constructor taking a raw pointer and taking ownership, the new version calls the move constructor